common/djangoapps/course_modes/tests/test_views.py

```diff
@@ -511,64 +516,118 @@ def test_course_closed(self):
             redirect_url = reverse('dashboard') + '?course_closed=1%2F1%2F15%2C+12%3A00+AM'
             self.assertRedirects(response, redirect_url)
 
-    # Value Prop TODO (REV-2378): remove waffle flag from tests once the new Track Selection template is rolled out.
-    # Other tests may need to be updated/removed to reflect the new page.
-    # The below test can be separated into multiple tests once un-happy path is implemented.
-    def test_new_track_selection(self):
-        # For the new track selection template to render, FBE must be fully on (gated_content and audit_access_deadline)
-        # and happy path conditions must be met:
-        # User can upgrade, FBE is fully on, and user is not an enterprise user.
+    def _assert_fbe_page(self, response, min_price=None, **_):
+        """
+        Assert fbe.html was transcluded.
```

Comment:
Would be beneficial to add a little more info on this description for those who are not familiar with what transcluded means.

Reply:
Sure!